lightning-c-bindings/src/routing/network_graph.rs

```diff
@@ -374,6 +376,17 @@ impl ChannelInfo {
 		ret
 	}
 }
+/// Protocol features of a channel communicated during its announcement
+#[no_mangle]
+pub extern "C" fn ChannelInfo_get_features(this_ptr: &ChannelInfo) -> crate::ln::features::ChannelFeatures {
```

Comment:
The Features stuff all uses `type X = Y` aliasing because they all use a common `Features<Context>` base, so I assume its because we started actually processing type aliasing now.